packages/migrate/migrations/svelte-5/migrate.js

```diff
@@ -29,6 +29,7 @@ export function update_pkg_json_content(content) {
 		['prettier', '^3.1.0'],
 		['prettier-plugin-svelte', '^3.2.6'],
 		['eslint-plugin-svelte', '^2.43.0'],
```

Comment:
the latest `eslint-plugin-svelte` uses the latest `svelte-eslint-parser`, so we should be fine if people are using it transitively
```suggestion
		['eslint-plugin-svelte', '^2.45.1'],
```